packages/faustwp-core/src/components/Toolbar/Toolbar.tsx

```diff
@@ -155,7 +155,10 @@ export function Toolbar({ seedNode }: ToolbarProps) {
         className="quicklinks"
         role="navigation"
         aria-label="Toolbar">
-        <ul id="wp-admin-bar-root-default" className="ab-top-menu">
+        <ul
+          id="wp-admin-bar-root-default"
+          className="ab-top-menu"
+          aria-label="primary-toolbar-nodes">
```

Comment:
```suggestion
          aria-label="primary toolbar menu items">
```

Reply:
Nice add!
This should use human readable language that a WordPress publisher might understand.